ai8x.py

```diff
@@ -33,6 +33,60 @@ def __call__(self, img):
         return img.sub(0.5).mul(256.).round().clamp(min=-128, max=127).div(128.)
 
 
+class fold:
+    """
+    Fold data to increase the number of channels. An interlaced approach used in this folding
+    as explained in [1].
+
+    [1] https://arxiv.org/pdf/2203.16528.pdf
+    """
+    def __init__(self, fold_ratio):
+        self.fold_ratio = fold_ratio
+
+    def __call__(self, img):
+        img_folded = None
```

Comment:
I think it'd be good to do nothing and return faster when fold_ratio==1. As it was implemented for the unfold.